flag-app/flag-app-project/src/components/CardWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import useCountries from "../hooks/useCountries";
+import Card from "./Card";
+import ErrorMessage from "./ErrorMessage";
+
+interface Props {
+  searchInput: string;
+}
+
+const CardWrapper = ({ searchInput }: Props) => {
+  const { countries, error, isLoading } = useCountries(searchInput);
+
+  if (isLoading) console.log(isLoading);
+  if (error) return <ErrorMessage searchInput={searchInput} />;
+
+  return (
+    <>
```

Comment:
This empty element is not needed, since there's nothing above it, and the `countries.map` is already wrapped by the `cards-layout` div.